cmd_states/run_test.py

```diff
@@ -0,0 +1,2 @@
+import Chandra.cmd_states
+Chandra.cmd_states.test()
```

Comment:
It seems like a reasonable requirement to have SKA set appropriately to build, but maybe we need to discuss?  I'm wondering how this plays in with potential automated remote builds.  And pardon my ignorance, but is conda-build running tests by default in the build process?  From the discussion that seems to be the case.  We can certainly skip tests that require SKA data or other particular interface requirements (like a .netrc file).